src/Installers/Windows/AspNetCoreModule-Setup/Directory.Build.props

```diff
@@ -70,5 +70,26 @@
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
       <SetPlatform>Platform=Win32</SetPlatform>
     </ProjectReference>
+   
+    <ProjectReference Include="$(_ServerIISBasePath)AspNetCoreModuleV2\AspNetCore\AspNetCore.vcxproj">
+      <Name>AspNetCoreV2ARM64</Name>
+      <Private>True</Private>
+      <DoNotHarvest>True</DoNotHarvest>
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+      <SetPlatform>Platform=ARM64</SetPlatform>
+    </ProjectReference>
+    <ProjectReference Include="$(_ServerIISBasePath)AspNetCoreModuleV2\OutOfProcessRequestHandler\OutOfProcessRequestHandler.vcxproj">
+      <Name>AspNetCoreV2HandlerARM64</Name>
+      <Private>True</Private>
+      <DoNotHarvest>True</DoNotHarvest>
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+      <SetPlatform>Platform=ARM64</SetPlatform>
+    </ProjectReference>
   </ItemGroup>
+
+  <Target Name="CopyANCM" BeforeTargets="_PrepareForBuild" Condition="!$(BuildNative)">
```

Comment:
Can you change the condition to `'$(BuildNative)' != 'true'` - we're moving away from the prettier boolean syntax since we can't use it everywhere & it isn't clear to readers why some conditions use it and others don't.